Meadowlark-js/backends/meadowlark-postgresql-backend/test/integration/Delete.test.ts

```diff
@@ -49,13 +46,6 @@ const newDeleteRequest = (): DeleteRequest => ({
   traceId: 'traceId',
 });
 
-const newGetRequest = (): GetRequest => ({
-  id: '',
-  resourceInfo: NoResourceInfo,
-  security: { ...newSecurity() },
-  traceId: 'traceId',
-});
```

Comment:
There were a couple of tests remaining from the initial implementation where I was trying to do Meadowlark gets rather that separate queries for validation